mmcls/datasets/voc.py

```diff
@@ -53,16 +55,29 @@ def load_annotations(self):
                 else:
                     labels.append(label)
 
-            gt_label = np.zeros(len(self.CLASSES))
+            gt_label = torch.zeros(len(self.CLASSES), dtype=torch.int8)
```

Comment:
Is it necessary to use tensor instead of array?

Reply:
Yes, it is the key. Only tensors are shared among dataloader workers by `torch.multiprocessing`. Numpy arrays, however, are deep copied to each workers, which consumes a lot of RAM.